server/pom.xml

```diff
@@ -205,6 +205,7 @@
                     <plugin>
                         <groupId>org.springframework.boot</groupId>
                         <artifactId>spring-boot-maven-plugin</artifactId>
+                        <version>2.4.5</version>
```

Comment:
Does this version can reuse `spring-boot.version` properties as same as spring boot version?

Reply:
done